src/test/java/com/datadog/api/TestUtils.java

```diff
@@ -136,8 +138,9 @@ public HttpURLConnection getConnection(URL url) throws IOException {
 
         protected static String TEST_API_KEY;
         protected static String TEST_APP_KEY;
+        protected static int WIREMOCK_PORT = 8080 + SUREFIRE_FORK - 1;
```

Comment:
why `-1`?

Reply:
Heh, I think this is a remainder of some other attempts and it's not important. I'll delete the `- 1` to make it less confusing.